tests/test_cv.py

```diff
@@ -177,3 +211,67 @@ def test_load_cv_folds_non_float_ids():
 
     # now read the CSV file using _load_cv_folds, which should raise ValueError
     _load_cv_folds(fold_file_path, ids_to_floats=True)
+
+def test_retrieve_cv_folds():
+    """
+    Test to make sure that the fold ids get returned correctly after cross-validation
+    """
+
+    learner = Learner('LogisticRegression')
+    num_folds = 5
+    cv_fs, custom_cv_folds = make_cv_folds_data(num_examples_per_fold=2, num_folds=num_folds)
+
+    expected_fold_ids = {'EXAMPLE_0': 0, 'EXAMPLE_1': 4, 'EXAMPLE_2': 3, 'EXAMPLE_3': 1,
```

Comment:
But that's what I'm testing here, that ``custom_cv_folds`` is actually being correctly returned from the function that computes it. I'm misunderstanding something?